packages/sdk/src/authentication/AADAuthProvider.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AxiosRequestConfig } from "axios";
+import { AppCredential, AuthenticationConfiguration } from "..";
+import { IAuthProvider } from "./IAuthProvider";
+
+export class AADAuthProvider implements IAuthProvider {
+  private appCredential: AppCredential;
+
+  constructor(config: AuthenticationConfiguration) {
```

Comment:
let support both client secret and obo scenario

Reply:
Fixed.